README.md

```diff
@@ -44,9 +44,7 @@ Wait until you receive an email approving your project!
 ### 4. Set the current project
 
 ```shell
-$ export CSCTL_CURRENT_PROJECT=`csctl projects list -o json | \
-  jq -r '.[] | select(.friendly_name == "sample-shopping-cart") | .name / "/" | .[1]'`
-$ csctl config set project $CSCTL_CURRENT_PROJECT
+$ csctl config set project 39ad1d96-466a-4d07-b826-b30509bda21b
```

Comment:
Are you removing this because it's easier to just paste the project name from the terminal than to have to install jq?

Reply:
Yes. I also think it's easier to paste the project ID than to find and paste this command. I don't think anyone will be able to remember it or come up with it off the top of his/her head.